### PR TITLE
Add openSendRequestForm action to prepare & navigate to the form

### PR DESCRIPTION
**Pull request by buoyad**
Requires #14326 

* Adds an action `openSendRequestForm` that takes all the possible `setBuilding*` params and dispatches the appropriate actions and load calls. Switches all places we nav to a send/request form to use that action.
* Makes `GetDisplayCurrencyLocal` param `accountID` optional and use the default accountID if none is given.

r? @keybase/react-hackers

### CODE REVIEW

File: protocol/avdl/stellar1/local.avdl [hunk 1]
```diff
@@ -201,7 +201,8 @@ protocol local {
   // OutsideCurrencyCode examples: "USD", "EUR". Has to be one of
   // supported currencies, returned from getDisplayCurrenciesLocal.
   void changeDisplayCurrencyLocal(int sessionID, AccountID accountID, OutsideCurrencyCode currency);
-  CurrencyLocal getDisplayCurrencyLocal(int sessionID, AccountID accountID);
+  // Gets display currency for primary account if no accountID is given.
+  CurrencyLocal getDisplayCurrencyLocal(int sessionID, union { null, AccountID } accountID);
```
Comment (zapu) [inline protocol/avdl/stellar1/local.avdl]: @patrickxb OK with API change?

Comment (patrickxb) [reply protocol/avdl/stellar1/local.avdl]: It seems fine, but just so I know, what path in the frontend needs to call this when it doesn't have an account id?

Comment (buoyad) [reply protocol/avdl/stellar1/local.avdl]: @patrickxb it's for prefilling the send form with the default display currency. When coming from chat / profile we haven't loaded any account IDs. When coming from the wallets tab we'll still pass in `accountID`

Comment (patrickxb) [reply protocol/avdl/stellar1/local.avdl]: Ok.  

Isn't GetWalletAccountsLocal() called pretty early on when the app starts up?  Then you would have the primary account ID.

Comment (buoyad) [reply protocol/avdl/stellar1/local.avdl]: We did briefly call that on startup but we don't anymore

**Comment by zanderz** [comment]
LGTM

**Pull request merged**